### PR TITLE
Add load state

### PR DESCRIPTION
**Pull request by zhouyuze**

### CODE REVIEW

**Comment by zhouyuze** [comment]
Fix Issue (#1)

File: src/main/java/cn/hashdata/bireme/LoadStatus.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
+package cn.hashdata.bireme;
+
+import java.util.HashMap;
+
+public class LoadStatus {
+  public HashMap<String, Long> produceTime;
+  public Long receiveTime = Long.MAX_VALUE;
+  public Long completeTime;
```
[the remark this answers is not in the document]
Comment (zhouyuze) [reply src/main/java/cn/hashdata/bireme/LoadStatus.java]: Sorry, forget to add comments.

File: src/main/java/cn/hashdata/bireme/LoadState.java [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+    return produceTime.get(table);
+  }
+
+  /**
+   * Reorganized the State into an object that is convenient to form a Json String.
+   * 
+   * @param targetTable the target Table
+   * @return A State object
+   */
+  public State toJson(String targetTable) {
```
Comment (wangzw) [inline src/main/java/cn/hashdata/bireme/LoadState.java]: better to rename toJson to getState

**Comment by wangzw** [comment]
LGTM except a small naming issue.

**Pull request merged**